Formula/c/checkov.rb

```diff
@@ -7,6 +7,12 @@ class Checkov < Formula
   sha256 "dca00fe8f0e04fd4fde88de5143bbe01aae7966351cd0f70f69456313bd43428"
   license "Apache-2.0"
 
+  livecheck do
+    url "https://pypi.org/rss/project/checkov/releases.xml"
+    regex(%r{<title>v?(\d+(?:\.\d+)+)</title>}i)
```

Comment:
```suggestion
    regex(/^v?(\d+(?:\.\d+)+)$/i)
    strategy :xml do |xml, regex|
      xml.get_elements("//item/title").map { |item| item.text[regex, 1] }
    end
```

Checking the PyPI RSS feed makes sense to me, though we should use the `Xml` strategy to make sure we're only checking `item` titles.